app/database/models.py

```diff
@@ -298,6 +299,16 @@ class Quote(Base):
     author = Column(String)
 
 
+class OutOfOffice(Base):
+    __tablename__ = "out_of_office"
+
+    id = Column(Integer, primary_key=True, index=True)
+    user_id = Column(Integer, ForeignKey("users.id"))
+    start_date = Column(DateTime, nullable=False)
+    end_date = Column(DateTime, nullable=False)
+    status = Column(String, nullable=False)
```

Comment:
sounds like a boolean (can can also set default value)

Reply:
I am using the column value in FE and BE  and not like boolean :)

Reply:
Which values do you have? I saw on (== True) and off (== False).
Do you have more valid values here?